<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>464</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>632</width>
    <height>464</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>3D Cart Brander and Emailer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="StartProgram">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>400</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start 3D Cart Brander</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Paste 3D Cart Email Here</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveTextToFile">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>310</y>
      <width>251</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Text to file: &quot;3dcart_info_Goes_Here.txt&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>236</x>
      <y>380</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Save before clicking this button</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="EmailTextBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>491</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearTextBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear TextBox</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>632</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
